cli/internal/publish/pluginui.go

```diff
@@ -120,14 +120,22 @@ func quickContentType(filename string) (string, error) {
 	}
 
 	contentType := http.DetectContentType(filebytes)
-	if contentType == "application/octet-stream" {
+	switch {
+	case contentType == "application/octet-stream":
```

Comment:
We can use the `mime` package similar to https://github.com/golang/go/issues/15888#issuecomment-222457018 WDYT?

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/19342/commits/a5579c5b3cfd96a495f3756dc62b78112ab7ffeb